scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,157 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-----------------------------------
+--Pre-requisite Completed Secrets of Oven Lost Log ID: 4, Quest ID: 73
+-----------------------------------
+-- Despachiaire !gotoid 16883721
+-- Rouva !gotoid 17719492
+--- Para NM ??? !gotoid 16785771
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+quest.reward =
+{
+    item     = xi.item.ELEGANT_RIBBON,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST)
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] = quest:progressEvent(517),
+
+            onEventFinish =
+            {
+                [517] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    npcUtil.giveKeyItem(player, xi.ki.ANTIQUE_AMULET)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED and vars.Prog <= 2
+        end,
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Rouva'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        return quest:progressEvent(747)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [747] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog',1)
+                end,
+            },
+        },
+
+        [xi.zone.CARPENTERS_LANDING] =
+        {
+            ['qm_para'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 1 and
+                        npcUtil.popFromQM(player, npc, ID.mob.PARA, { claim = true, hide = 0 })
+                    then
+                        return quest:messageSpecial(ID.text.STENCH_OF_DECAY)
+                    elseif
+                        quest:getVar(player, 'Prog') == 2 and
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        npcUtil.giveKeyItem(player, xi.ki.CATHEDRAL_MEDALLION)
+                        quest:setVar(player, 'prog', 3)
+                        player:delKeyItem(xi.ki.ANTIQUE_AMULET)
```

Comment:
I don't know the quest, is this silent, or does it produce a message, should it?

Reply:
It should produce a message and does.
![xiloader_KQmhzib8yj](https://github.com/user-attachments/assets/ef7bd83a-8d7b-435f-9254-d1f0b6a0c297)

Reply:
Ideally, the messages should be swapped to match the retail capture. However, no matter what I tried I couldn't get them to swap, and both display.

Reply:
That was tested and didn't work. The way it is now is the only way I could get it to work where both lines appear.  Having the quest flavor text display is more important than swapping the order IMO.

From testing after reordering:
![xiloader_ixPfgcmPvy](https://github.com/user-attachments/assets/065828a0-0f75-4ff7-be65-bba0a04d823f)

Reply:
![image](https://github.com/user-attachments/assets/b536d0c2-6ee5-4fbf-9d1c-a103f2d42814)
It looks like player:messageSpecial should work? not sure what quest:messageSpecial is doing differently

Reply:
![image](https://github.com/user-attachments/assets/e9ecaadf-7fbe-481c-acec-1025591d1e99)
hmm... the "nothing out of the ordinary" is probably from not having a return value

Reply:
![image](https://github.com/user-attachments/assets/c3d9b51b-8fd0-4484-9063-eb37d6a5da6f)
ok, this works but I know i've heard Clay say that `noAction` is very rarely used and there might be a different way

```
                    then
                        quest:setVar(player, 'prog', 3)
                        player:delKeyItem(xi.ki.ANTIQUE_AMULET)
                        player:messageSpecial(ID.text.POLISH_MUSHROOM_SPORE, xi.ki.ANTIQUE_AMULET)
                        npcUtil.giveKeyItem(player, xi.ki.CATHEDRAL_MEDALLION)

                        return quest:noAction()
                    end
```

Reply:
```lua
                    then
                        quest:setVar(player, 'prog', 3)
                        player:delKeyItem(xi.ki.ANTIQUE_AMULET)
                        player:addKeyItem(xi.ki.CATHEDRAL_MEDALLION)

                        player:messageSpecial(ID.text.POLISH_MUSHROOM_SPORE, xi.ki.ANTIQUE_AMULET)
                        return quest:messageSpecial(ID.text.KEYITEM_OBTAINED, xi.ki.CATHEDRAL_MEDALLION)
                    end
```

This works
![image](https://github.com/user-attachments/assets/fbb980e8-d2d0-4c97-897e-ca347aaacae2)

Reply:
Confirmed that works like a charm. Thank you for the assist @WinterSolstice8